pkgs/applications/misc/bukut/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, python3, fetchFromGitHub }:
+
+with python3.pkgs; buildPythonApplication rec {
+  pname = "bukut";
+  version = "0.11";
+
+  src = fetchFromGitHub {
+    owner = "peterjschroeder";
+    repo = "bukut";
+    # project doesn't have tags
+    rev = "9d735490dae9f657a74f33f2eb68656b31a5c5d8";
+    sha256 = "sha256-MCAyMvzXnC7QSpWeG7qorK4JmRBMzG3eGsp5YpKmVjg=";
+  };
+
+  propagatedBuildInputs = [
+    asciimatics
+    beautifulsoup4
+    natsort
+    pyperclip
+    pyxdg
+  ];
+
+  meta = with lib; {
+    description = "Text user interface for buku bookmark manager";
+    homepage = "https://github.com/peterjschroeder/bukut";
```

Comment:
Missing license attribute

Reply:
The project itself doesn't have a license. I can add a comment but the attribute itself seems to be optional.

Reply:
If a project doesn't have a license, it's `unfree` by default

Reply:
https://github.com/peterjschroeder/bukut/issues/2 I've opened an issue asking the author to clarify